scripts/nayduck.py

```diff
@@ -41,9 +41,12 @@ def _parse_args():
     parser.add_argument('--branch',
                         '-b',
                         help='Branch to test. By default gets current one.')
-    parser.add_argument('--sha',
-                        '-s',
-                        help='Sha to test. By default gets current one.')
+    parser.add_argument(
+        '--sha',
+        '-s',
+        help=
+        'Commit sha to test. By default gets current one. This is ignored if branch name is provided'
```

Comment:
for a bit more safety you can actually bail out and log an error if both sha and branch are provided